src/settings.js

```diff
@@ -1999,6 +1999,16 @@ var TRUSTED_TYPES = false;
 // settings is *only* needed when also explicitly targeting older browsers.
 var POLYFILL = true;
 
+// If true, add tracing to core runtime functions.
+// This settings is enabled by default is any of the other debugging settings
```

Comment:
```suggestion
// This setting is enabled by default if any of the following debugging settings
```

Reply:
Would you prefer for me to simply commit such typo changes, then approve? (then automerge would land it)

Reply:
Either way is fine with me.  I'm also OK with an "lgtm with comments addressed" (Apologies for so many typos in my PRs.. grammar and spelling were never my strong suites).